### PR TITLE
Add conference support

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -93,6 +126,40 @@ impl Event {
                     friend,
                     message_id,
                 },
+
+            E::ConferenceInvite { friend, kind, ref cookie } =>
```
Comment (kpp) [inline src/lib.rs:149]: Why `ref cookie`?

Comment (suhr) [reply src/lib.rs:149]: `Cookie` is a `Vec<u8>` inside.

Comment (kpp) [reply src/lib.rs:149]: And then you do `cookie.clone().into_bytes()`, but you could do `cookie.into_bytes()` without `ref cookie`

Comment (suhr) [reply src/lib.rs:149]: `into_bytes()` moves `cookie`

Comment (kpp) [reply src/lib.rs:149]: > `match *event {`

I see.

**Pull request merged**